.github/workflows/build_and_run.yml

```diff
@@ -68,7 +68,7 @@ jobs:
           ls $(dirname $(dirname `which icx`))/bin-llvm || exit 1
           export NUMBA_DPEX_LLVM_SPIRV_ROOT=$(dirname $(dirname `which icx`))/bin-llvm
 
-          python -c "import dpbench; dpbench.run_benchmarks()" || exit 1
+          python -c "import dpbench; dpbench.run_benchmarks(repeat=2)" || exit 1
```

Comment:
Maybe we should just make `repeat=1`? All we are currently doing is just validation.

Reply:
I just wanna be sure, that it actually can be repeatable. That's the reason for 2.

Reply:
The time difference for this step is not that dramatic 6:20s vs 7:40s.